### ISSUE TITLE
Colons in JS includes

### ISSUE THREAD

**Issue by tcmb** [johaness/packular#6]
I'm trying to include a script from a URL which contains a colon as a port specifier, with a line in the `[javascript]` section like `example.com:8080/awsm.js = dev,prod `.

The colon gets interpreted by `ConfigParser` as a key/value separator and the include doesn't work.

Suggestion:
I read that the Python 3 version of configparser lets you specify the k/v separator, and that there is a backport to Python 2.7 (https://pypi.python.org/pypi/configparser). But I'm aware that this would break the 'no dependencies' part of packular.

**Comment by johaness** [comment]
Ja, we really like the /no dependencies/ bit.

Maybe try overwriting the regex in ``DefaultConfigParser``, eg (untested!):
```
OPTCRE = re.compile(ConfigParser.OPTCRE.pattern.replace(':', ''), ConfigParser.OPTCRE.flags)
```
looks like it could work in both Pythons

**Comment by tcmb** [comment]
Solved via #7
